api/modules/food/constants.py

```diff
@@ -0,0 +1,11 @@
+"""
+This has used the Zomato Api
+For more information look into https://developers.zomato.com/api
+"""
+import os
+
+ORIGINAL_ZOMATO_API = "9a656813579b698902619a05684d39c6"
```

Comment:
Don't mention API Key here
Do local testing using the key, but do not commit the key in file

Reply:
But I followed what was done in the weather API thing

Reply:
Oh my bad, I will remove the key from there as well 
Thanks for pointing out 👍

Reply:
So where does the API go?

Reply:
As i told earlier I will set an environment variable for actual server
For local you can test it using this key but do not add it to the code